javascript-objects/main.js

```diff
@@ -0,0 +1,38 @@
+const student = {
+  firstName: 'Kasiet',
+  lastName: 'Rakhimdinova',
+  age: '21'
+};
+const fullName = student.firstName + ' ' + student.lastName;
+
+console.log('Value of fullName ==> ', fullName);
+
+student.livesInIrvine = true;
+student.previousOccupation = 'Orthodontist';
+
+console.log('Value of livesInIrvine ==>', student.livesInIrvine);
+console.log('Value of previousOccupation ==>', student.previousOccupation);
+console.log('Value of student object ==>', student);
+
+const vehicle = {
+  make: 'Toyota',
+  model: 'Camry',
+  year: 2021
+};
+
+vehicle['color'] = 'Space teal';
+vehicle['isReliable'] = true;
```

Comment:
This should be `isConvertible` rather than `isReliable`. Although I agree that Camry's are very reliable 😆